change_color.sh

```diff
@@ -134,7 +134,7 @@ done
 
 cd "$DEST_PATH"
 for FILEPATH in "${PATHLIST[@]}"; do
-	find $(echo "${FILEPATH}" | sed 's/src\///g' ) -type f -exec sed -i \
+	find $(echo "${FILEPATH}" | sed -e 's/src\///g' ) -type f -exec sed -i.bak \
```

Comment:
if you creating those files you need to clean up them afterwards, otherwise they will persist to the generated theme